contracts/light-clients/ics02-wrapper/README.md

```diff
@@ -0,0 +1,12 @@
+# ICS-02 Precompile Wrapper
+
+This solidity IBC light client contract serves as a wrapper for the [ICS02 Precompile](https://github.com/cosmos/evm/tree/main/precompiles/ics02) in [`cosmos/evm`](https://github.com/cosmos/evm/).
+
+## Overview
+
+This contract, when deployed on a Cosmos SDK-based blockchain with EVM support, allows users to interact with a IBC-Go light client through a familiar Solidity interface. This contract is used to integrate `solidity-ibc-eureka` with Cosmos SDK-based chains that support the ICS02 precompile.
```

Comment:
Would be good to state why we need a wrapper - I almost forgot that it was to allow us to have different addresses for each light client until I remembered our conversation.